### ISSUE TITLE
Bulk returns "something went wrong error"

### ISSUE THREAD

**Issue by betsydupuis** [public-accountability/littlesis-rails#1695]
Bulk add returns an error.

<img width="475" alt="Screenshot 2024-06-19 at 1 50 52 PM" src="https://github.com/public-accountability/littlesis-rails/assets/2096627/5073998e-9454-444e-9344-cd62d08362eb">

**Comment by josephlacey** [comment]
Hi @betsydupuis , Thank you for the report.  Do you have any more details about the data you were entering when the error happened?

**Comment by betsydupuis** [comment]
No details other than I noticed this first happened about a month ago.

**Comment by betsydupuis** [comment]
As far as the data goes, it’s been happening on multiple different
configurations of trying to use the bulk feature.

**Comment by betsydupuis** [comment]
I couldn't see anything in the console either. 

If you can't reproduce it, perhaps you can console.log() the error message  in the next build, and I can send it to you.

**Comment by josephlacey** [comment]
@betsydupuis We've had a ongoing issue with the bulk upload feature and have a medium term plan to rewrite it.  The only reliable option at the moment is for me to send over a CSV template and you send it back with the data you want to upload.  I can then import it from the command line.  Should I send that over?

**Comment by betsydupuis** [comment]
Yes, please send it over. I don't use bulk upload that often, but I would probably use it more if CSV was an option.

**Comment by josephlacey** [comment]
> Yes, please send it over. I don't use bulk upload that often, but I would probably use it more if CSV was an option.

Here's the basic template.  If you have other fields you want to include, let me know and we can make some adjustments.
[littlesis-import-template.csv](https://github.com/user-attachments/files/15993618/littlesis-import-template.csv)

**Comment by betsydupuis** [comment]
Thanks!

I'm a UX designer and I've worked on similar systems to Little Sis in the past. I'd love to help out if you have room for it.

I also know Angular 1 and React.

**Comment by josephlacey** [comment]
> I'm a UX designer and I've worked on similar systems to Little Sis in the past. I'd love to help out if you have room for it.

Wow, that's great.  We actually might have a short term need for some improvements from a UX designer.  Would you be available to have a quick chat in the next few days?  Why don't we follow up outside of GitHub on this conversation, joseph@littlesis.org.
 
> I also know Angular 1 and React.

Just a quick note that our mapping tool is built in React,

https://github.com/public-accountability/oligrapher

We have occasional bugs that come up plus a list of pending features we want to put some work into.
